learning_resources/models.py

```diff
@@ -17,6 +18,7 @@ class LearningResourcePlatform(TimestampedModel):
     """Platforms for all learning resources"""
 
     platform = models.CharField(max_length=12, primary_key=True)
```

Comment:
Should this be platform_code?

Reply:
That's a better field name, but it probably should be done in a separate PR because it's going to require changing lots of files.  I can create a new issue for it.

Reply:
https://github.com/mitodl/mit-open/issues/113